app/config.go

```diff
@@ -810,7 +856,8 @@ func ResetAll(dbDir, addrBookFile, privValKeyFile, privValStateFile string, logg
 	// recreate the dbDir since the privVal state needs to live there
 	err := cmn.EnsureDir(dbDir, 0700)
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
```suggestion
              log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work